scripts/webpack.config.babel.js

```diff
@@ -0,0 +1,173 @@
+import path from "path"
+import webpack from "webpack"
+import ExtractTextPlugin from "extract-text-webpack-plugin"
+
+import pkg from "../package.json"
+import config from "./config.js"
+
+export default {
+  module: {
+    loaders: [
+      { // statinamic requirement
+        test: /\.md$/,
+        loader: "statinamic/lib/md-collection-loader" +
+          `?${ JSON.stringify({
```

Comment:
I will do that later: this entire json.stringshit will be (re)moved anyway in a future statinamic upgrade.